examples/widgets/list/lv_example_list_2.c

```diff
@@ -95,17 +95,15 @@ static void event_handler_bottom(lv_event_t * e)
 static void event_handler_swap(lv_event_t * e)
 {
     const lv_event_code_t code = lv_event_get_code(e);
-    // lv_obj_t* obj = lv_event_get_target(e);
     if((code == LV_EVENT_CLICKED) || (code == LV_EVENT_LONG_PRESSED_REPEAT)) {
         uint32_t cnt = lv_obj_get_child_count(list1);
-        for(int i = 0; i < 100; i++)
-            if(cnt > 1) {
-                lv_obj_t * obj = lv_obj_get_child(list1, (int32_t)lv_rand(0, cnt));
-                lv_obj_move_to_index(obj, (int32_t)lv_rand(0, cnt));
-                if(currentButton != NULL) {
-                    lv_obj_scroll_to_view(currentButton, LV_ANIM_ON);
-                }
+        for(int i = 0; i < 100; i++) {
+            lv_obj_t * obj = lv_obj_get_child(list1, (int32_t)lv_rand(0, cnt - 1));
+            lv_obj_move_to_index(obj, (int32_t)lv_rand(0, cnt - 1));
+            if(currentButton != NULL) {
```

Comment:
IMO we should move it out of the loop to avoid scrlloing 100 times
```suggestion
            if(currentButton != NULL) {
```